operate/client/e2e-playwright/tests/childProcessInstanceMigration.spec.ts

```diff
@@ -0,0 +1,204 @@
+    ).toContainText(/2 results/i);
+
+    // Navigate to parent process instance page
+    await processesPage.processInstancesTable
+      .getByRole('link', {
+        name: parentInstanceKey,
+      })
+      .click();
+
+    // Click on button to see called elements (child processes) in process instance list
+    await processInstancePage.instanceHeader
+      .getByRole('link', {
+        name: 'View all',
+      })
+      .click();
+
+    // Get child process version that was called by parent process instance
+    const childVersion = await processesPage.processInstancesTable
+      .getByTestId('cell-processVersion')
+      .innerText();
+
+    // Check if parent process instance key is set (filtering by parent instance key is on here)
+    await expect(
+      processesPage.processInstancesTable.getByRole('cell', {
+        name: parentInstanceKey,
+      }),
+    ).toBeVisible();
+
+    // Select child process and version before going into migration
+    await processesPage.navigateToProcesses({
+      searchParams: {
+        active: 'true',
+        process: childBpmnProcessId,
+        version: childVersion,
+        parentInstanceId: parentInstanceKey,
+      },
+    });
+
+    await expect(
+      processesPage.processInstancesTable.getByText(/1 result/i),
+    ).toBeVisible();
+
+    // Select first child process
+    await processesPage.getNthProcessInstanceCheckbox(0).click();
+    await expect(page.getByText(/1 item selected/i)).toBeVisible();
+
+    // Check if migrate button is enabled
+    await expect(processesPage.migrateButton).toBeEnabled();
+
+    // Go into process instance migration
+    await processesPage.migrateButton.click();
+    await processesPage.migrationModal.confirmButton.click();
+
+    // Select target process, version and map flow nodes
+    await migrationView.selectTargetProcess('childProcess');
+    await migrationView.selectTargetVersion(targetVersion);
+    await migrationView.mapFlowNode({
+      sourceFlowNodeName: 'New Task',
+      targetFlowNodeName: 'Task',
+    });
+
+    // Confirm and finish migration
+    await migrationView.nextButton.click();
+    await expect(migrationView.summaryNotification).toContainText(
+      `You are about to migrate 1 process instance from the process definition: ${childBpmnProcessId} - version ${childVersion} to the process definition: ${childBpmnProcessId} - version ${targetVersion}`,
+    );
+    await migrationView.confirmButton.click();
+
+    await expect(commonPage.operationsList).toBeVisible();
+
+    const migrateOperationEntry = commonPage.operationsList
+      .getByRole('listitem')
+      .first();
+
+    await expect(migrateOperationEntry).toContainText('Migrate');
+
+    await expect(migrateOperationEntry.getByRole('progressbar')).toBeVisible();
+
+    // wait for migrate operation to finish
+    await expect(
+      migrateOperationEntry.getByRole('progressbar'),
+    ).not.toBeVisible();
```

Comment:
This is the point where the roundtrip happens (we're waiting for Zeebe to finish and for Operate to import the changes).

I saw the other migration test failing often in CI at this point, because the Operation had not been finished before the timeout (which I think is 30 seconds?). So I think it makes sense to increase the timeout here (and in processInstanceMigration.spec.ts too):

```suggestion
    await expect(
      migrateOperationEntry.getByRole('progressbar'),
    ).not.toBeVisible([timeout: 60000);
```

This will hopefully increase the stability a bit.

Reply:
This makes sense to me, I'll make the change in these 2 tests and also on `batchMoveModification`, that has an identical line and has already been flaky before too.